systemtest/src/test/java/io/strimzi/systemtest/tracing/OpenTelemetryTracingST.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.tracing;
+
+import io.strimzi.api.kafka.model.tracing.JaegerOTelTracing;
+import io.strimzi.api.kafka.model.tracing.Tracing;
+import io.strimzi.systemtest.annotations.IsolatedSuite;
+
+@IsolatedSuite
+public class OpenTelemetryTracingST extends TracingBaseST {
+    @Override
+    protected Tracing tracing() {
+        return new JaegerOTelTracing();
+    }
+
+    protected String serviceNameEnvVar() {
+        return "OTEL_SERVICE_NAME";
+    }
+}
```

Comment:
I don't really like the idea of just extending the `TracingBaseST`, without the separated methods. If you want to execute `OpenTelemetryTracingST` test, just a single test, like `testProducerConsumerStreamsService`, in the IDE, how will you do that?

It would be much easier to read/execute it if in the `TracingBaseST` you would have the tests itself (maybe rename it to `dotestProducerConsumerStreamsService` would make sense) and in the `OpenTelemetryTracingST` you will have tests which will point to the `TracingBaseST`. Look at the `OlmAbstractST`,`AllNamespacesST` and `SingleNamespaceST` inside the `olm` package. That's the way how it should be done.

Reply:
I fail to see why this is better.
Just the opposite, more work for no gain?

Reply:
> If you want to execute OpenTelemetryTracingST test, just a single test, like testProducerConsumerStreamsService, in the IDE, how will you do that?

In IntelliJ this is trivial -- just click on the `play` next to the test method (in the base / abstract class), and it asks you what you wanna run -- single explicit test (on non-abstract class) or all impls

Reply:
FMPOV it would be much clearer to have it like in the STs I mentioned above. I know that it is much easier to do this the way you have done, but to make it clear (also for the newcomers or anyone who don't know how it works), it should be done the second way. WDYT @Frawless ?

Reply:
OK, changed to match existing STs.

Reply:
I agree with @im-konge